cg/store/api/find_business_data.py

```diff
@@ -513,6 +513,22 @@ def get_samples_by_subject_id_and_is_tumour(
                 samples=samples, filter_functions=[SampleFilter.FILTER_IS_NOT_TUMOUR]
             ).all()
 
+    def get_samples_by_customer_id_list_and_subject_id_and_is_tumour(
+        self, customer_ids: List[int], subject_id: str
+    ) -> List[Sample]:
+        """Return a list of samples matching a list of customers with given subject_id and is_tumour."""
+        samples = self._get_query(table=Sample)
+        return apply_sample_filter(
+            samples=samples,
+            customer_ids=customer_ids,
+            subject_id=subject_id,
+            filter_functions=[
+                SampleFilter.FILTER_BY_CUSTOMER_ID,
```

Comment:
This name is misleading since it is in singular, I would prefer adding a separate filter for filtering on multiple customer id:s. Alternatively renaming the SampleFilter.FILTER_BY_CUSTOMER_ID to FILTER_BY_CUSTOMER_IDS.

Reply:
I am addressing this in another PR :)